week1-section3/exercise 5/Main.cpp

```diff
@@ -0,0 +1,46 @@
+/*Exercise 5 - Arrays
+Create an array of whatever size you want. Let the user input as many numbers as they want and populate the array with them. If they enter any invalid input, output the whole array of numbers and the sum of them all and the total numbers they input.
+
+If the user exceeds the size of your array, tell them about it and then output the information.
+*/
+
+#include <iostream>
+
+
+
+
+int main(){
+
+	int numbers[10];
+	int  insert, sum=0;
+	int size = 0;
+
+
+
+	for ( int i = 0; i < 10; i++){
+		std::cout << "insert any NUMBER: ";
+		std::cin >> insert;
+		sum = sum + insert;
+		insert = numbers[i];
```

Comment:
Remember that you want to have the variable you want to assign a value to BEFORE the `=` and the value you want to assign to it AFTER the `=`. 

This should be written as `numbers[i] = insert;`